package.json

```diff
@@ -171,7 +171,7 @@
 		"@wordpress/e2e-test-utils-playwright": "0.6.0",
 		"@wordpress/e2e-tests": "^4.6.0",
 		"@wordpress/element": "4.20.0",
-		"@wordpress/env": "7.0.0",
+		"@wordpress/env": "8.1.1",
```

Comment:
What do you think if instead of `8.1.1`, we could use `wp-6.2`?

My idea is to follow this approach: https://github.com/woocommerce/woocommerce-blocks/pull/10331/files